Gibe.DittoProcessors/Processors/CanonicalUrlAttribute.cs

```diff
@@ -0,0 +1,12 @@
+using Umbraco.Core;
+using Umbraco.Web;
+
+namespace Gibe.DittoProcessors.Processors
+{
+	public class CanonicalUrlAttribute : TestableDittoProcessorAttribute
+	{
+		public override object ProcessValue() => Context.Content.DocumentTypeAlias.InvariantEquals("home")
```

Comment:
Are we always going to call our homepage doctypes "home"?

Reply:
Not sure, its just the same processor we copy and use everywhere, but we can do!

Reply:
this'll get the wrong url for the homepage if we don't call it "home", though to be honest I don't really feel like this is the right place to figure that url out anyway - it'd still be wrong in pickers and links and such! 

So long as we use `home`, this shouldn't _break_, though, at least not obviously.

Reply:
It maybe possible to have multiple homepages setup and one of them chosen as the actual current homepage, would all of these definitely have a doc type alias of "home"?

This feels pretty fragile to me, there is more of a convention for selecting the homepage from the root node using umbracoInternalRedirectId so maybe we should say is the current content the same as what is selected in umbracoInternalRedirectId on the root node?

Reply:
It is fragile, I was just tired of copying this work by another dev between projects - feel free to improve!